eva/parser/table_ref.py

```diff
@@ -252,3 +252,104 @@ def __eq__(self, other):
 
     def __hash__(self) -> int:
         return hash((self._ref_handle, self.alias, self.sample_freq, self.sample_type))
+
+
+class AVTableRef(TableRef):
```

Comment:
While you are working on this rewrite, also think if we can improve `TableRef` class. For example, can we remove unnecessary complexity? Thanks!